pages/en/lb3/Storage-component.md

```diff
@@ -247,7 +249,31 @@ as shown in the following table.
       <td>authUrl</td>
       <td>Your identity service</td>
     </tr>
-
+    <tr>
+      <td rowspan="4"><strong>Google Cloud</strong></td>
+      <td>provider</td>
+      <td>'google'</td>
+      <td rowspan="4">
+<pre style="font-size:11px;">{
+ provider: 'google',
+ 'keyFilename': 'path/to/keyfile.json',
+ 'projectId': '...',
+ 'nameConflict': 'makeUnique'
+}</pre>
+      </td>
+    </tr>
+    <tr>
+      <td>keyFilename</td>
+      <td>Path to key file.</td>
+    </tr>
+    <tr>
+      <td>projectID</td>
+      <td>Google Cloud project ID.</td>
+    </tr>
+    <tr>
+      <td>nameConflict</td>
```

Comment:
Ah, right.  It is actually documented here https://loopback.io/doc/en/lb3/Storage-component.html#automatic-unique-filenames.

Since it's not unique to Google, I'll remove it from the table entry.